modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-de_DE.json

```diff
@@ -245,7 +245,14 @@
       "ADOPTER_PROFILE_UPDATED": "Profil-Informationen erfolgreich aktualisiert.",
       "ADOPTER_PROFILE_NOT_UPDATED": "Hoppla! Profil-Informationen konnten nicht aktualisiert werden. Bitte versuchen Sie es erneut!",
       "ADOPTER_PROFILE_DELETED": "Profil-Informationen erfolgreich gelöscht.",
-      "ADOPTER_PROFILE_NOT_DELETED": "Hoppla! Profil-Informationen konnten nicht gelöscht werden. Bitte versuchen Sie es erneut!"
+      "ADOPTER_PROFILE_NOT_DELETED": "Hoppla! Profil-Informationen konnten nicht gelöscht werden. Bitte versuchen Sie es erneut!",
+      "TOBIRA_NO_TITLE": "Seite benötigt einen Titel",
+      "TOBIRA_NO_PATH_SEGMENT": "Seite benötigt ein Pfadsegment",
+      "TOBIRA_NOT_MOUNTED": "Serie konnte nicht in Tobira eingehangen werden",
+      "TOBIRA_PATH_SEGMENT_INVALID": "Pfadsegment ist ungültig. Ein gültiges Pfadsegment ist mindestens zwei Zeichen lang, enthält keine Leerzeichen, Kontrollzeichen, und keine der Zeichen '<>\"[\\]^`{|}#%\/?]', und fängt mit keinem der Zeichen '-+~@_!$&;:.,=*'()' an.",
+      "TOBIRA_PATH_SEGMENT_UNIQUE": "Das gleiche Pfadsegment darf nicht mehrmals für Seiten mit der gleichen Überseite verwendet werden.",
```

Comment:
Some validation messages have punctuation, others don't. Should we unify that? 🤔